scripts/clean.sh

```diff
@@ -0,0 +1,4 @@
+rm -rf packages/views/lib
```

Comment:
Codacy found an issue: [Tips depend on target shell and yours is unknown. Add a shebang.](https://app.codacy.com/manual/yajananrao/Nadha/pullRequest?prid=5885240)